test/test_helper.rb

```diff
@@ -0,0 +1,21 @@
+root_location = File.expand_path("../../", __FILE__)
+ENV["Y2DIR"] = File.expand_path("../../src", __FILE__)
+
+require "yast"
+require "fileutils"
+
+if ENV["COVERAGE"]
+  require "simplecov"
+  SimpleCov.start
+  # for coverage we need to load all ruby files
+  Dir["#{root_location}/library/*/src/{module,lib}/**/*.rb"].each { |f| require_relative f }
```

Comment:
I don't think this fits the structure of yast-autoinstallation. Looks more like copipasted from yast-yast2, which has an special file organization.